homeassistant/components/zwave_js/__init__.py

```diff
@@ -103,6 +103,10 @@ def register_node_in_dev_reg(
 
     async_dispatcher_send(hass, EVENT_DEVICE_ADDED_TO_REGISTRY, device)
 
+    # We can assert here because we will always get a device
+    assert device
```

Comment:
We should update `get_or_create` in a new PR and remove that it can return `None`, that's just weird.

Reply:
it's because of this, but shouldn't we just raise an exception?
https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/device_registry.py#L264-L265

Reply:
https://github.com/home-assistant/core/pull/49038

Reply:
We should be able to remove this now if we rebase.

Reply:
We can do this in a separate PR.